tools/hangar/__snapshots__/test_corpus/valid/bring_wing_library.test.w_compile_tf-aws.md

```diff
@@ -186,7 +186,7 @@ module.exports = function({ $stdlib }) {
       this.data = this.node.root.new("@winglang/sdk.cloud.Bucket", cloud.Bucket, this, "cloud.Bucket");
     }
     static makeKey(name) {
-      return (require("<ABSOLUTE_PATH>/util.js")["makeKey"])(name)
+      return (require("../../../node_modules/@winglibs/testfixture/util.js")["makeKey"])(name)
```

Comment:
I'll probably do a followup to emit this as `require("@winglibs/testfixture/util.js")` instead

Reply:
nvm I did it now